react/components/EditorProvider.tsx

```diff
@@ -19,7 +19,9 @@ import MessagesContext, { MessagesContextProps } from './MessagesContext'
 
 type Props = RenderContextProps &
   ReactIntl.InjectedIntlProps &
-  MessagesContextProps & { client: ApolloClient<unknown> }
+  MessagesContextProps & { client: ApolloClient<unknown> } & {
```

Comment:
⛏ `{ client: ApolloClient<unknown>, isSiteEditor: boolean }`

Reply:
Oops, totally missed that! Nice catch!